paradise_papers_django/search/templates/search/nodes_templates/entity.html

```diff
@@ -6,27 +6,26 @@
         <div  class="node-field"> Status </div>
         <div class="node-field"> Data From </div>
     </div>
-
     {% for entity in entities %}
         <div class="column-content">
             <div class="node-title">
-                <a href="/search/{{ entity.node_id }}">{{entity.name}}</a>
+                <a href="/search/{{ node_type }}/{{ entity.node_id }}">{{ entity.name }}</a>
```

Comment:
This can be a solution to the dynamic url issue but this is a context variable and won't change when you go to others entities